contrib/opencensus-ext-azure/opencensus/ext/azure/common/__init__.py

```diff
@@ -17,14 +17,73 @@
 
 from opencensus.ext.azure.common.protocol import BaseObject
 
+AUTHORIZATION = 'Authorization'
+DEFAULT_BREEZE_ENDPOINT = 'https://dc.services.visualstudio.com'
+ENDPOINT_SUFFIX = 'EndpointSuffix'
+ENV_CONNECTION_STRING = 'APPLICATIONINSIGHTS_CONNECTION_STRING'
+ENV_INSTRUMENTATION_KEY = 'APPINSIGHTS_INSTRUMENTATIONKEY'
```

Comment:
If this is going to be used once, why do we want to define it here?
Putting the literal string in the code improves readability as one wouldn't have to jump to different places to read the code.